api/k8s/service/values.yaml

```diff
@@ -129,42 +129,41 @@ configMap:
   create: true
   name: ""
   annotations: {}
-  data:
-    application.yaml: |
-      spring:
-        cloud:
-          azure:
-            profile:
-              tenant-id: ${AZURE_TENANT_ID}
-            credential:
-              client-id: ${AZURE_CLIENT_ID}
-              client-secret: ${AZURE_CLIENT_SECRET}
-      enums:
-        escalationTeams:
-          - label: core
-            code: core
-            slack-group-id: S08948NBMED
-      platform-integration:
-        enabled: true
+  config:
+    spring:
+      cloud:
         azure:
-          enabled: true
-        static-user:
+          profile:
+            tenant-id: ${AZURE_TENANT_ID}
+          credential:
+            client-id: ${AZURE_CLIENT_ID}
+            client-secret: ${AZURE_CLIENT_SECRET}
+    enums:
+      escalationTeams:
+        - label: core
+          code: core
+          slack-group-id: S08948NBMED
+    platform-integration:
+      enabled: true
+      azure:
+        enabled: true
+      static-user:
+        enabled: false
+      teams-scraping:
+        static:
           enabled: false
-        teams-scraping:
-          static:
-            enabled: false
-          core-platform:
-            enabled: false
-          k8s-generic:
-            enabled: true
-            config:
-              apiVersion: v1
-              kind: ConfigMap
-              namespace: support-bot-integration
-              filter:
-                name-regexp: ^tenant-.*
-                label-selector: null
-              teamName:
-                pointer: /data/name
-              groupRef:
-                pointer: /data/groupRef
+        core-platform:
+          enabled: false
+        k8s-generic:
+          enabled: true
+          config:
+            apiVersion: v1
+            kind: ConfigMap
+            namespace: support-bot-integration
```

Comment:
QQ: why is the namespace got a integration suffix? Is it's main purpose for running integrations tests? 

Straight away when I look at this config I think it's deploying the support bot into this namespace so why can't it just be `support-bot`?